migrations/blocked_tasks.js

```diff
@@ -0,0 +1,74 @@
+function taskIsBlocked(task) {
+    for(dep of task.depends_on) {
+        if (dep.unattainable) {
+            return true
+        }
+    }
+
+    return false
+}
+
+function unattainableResolved(task) {
+    for (dep of task.depends_on) {
+        if (dep.unattainable == null) {
+            return false
+        }
+    }
+    return true
+}
+
+finishedStatuses = ["success","failed"]
+millisecondsPerDay = 86400000
+delta = millisecondsPerDay * 1
+
+startingDate = ISODate("2016-01-01T00:00:00Z")
+while(startingDate < new Date()) {
```

Comment:
I think it'd be useful to encapsulate this in a function with number of loops, delta, and the argument to sleep as parameters. Then, instead of executing this file at the shell executing the migration, you can execute this file only to load the function, and then run the function from the shell. This would allow you to tune things more quickly if, for example, we find that we want to put more or less load on the db.

Reply:
done.